pom.xml

```diff
@@ -0,0 +1,621 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <name>Closure Templates</name>
+  <groupId>com.google.template</groupId>
+  <artifactId>soy</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
I agree with using `2.5.0-SNAPSHOT`, seems the most accurate. I think the transitive dependency issue you noted is unlikely and could be resolved with exclusions.